### DISCUSSION TITLE
Welcome to Virtual-Display-Driver Discussions!

### DISCUSSION THREAD

**Discussion by itsmikethetech** [VirtualDrivers/Virtual-Display-Driver#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DandamanAUS** [comment]
Getting Error C1083 Cannot open include file: 'iddcx.h': No such file or directory IddSampleDriver.

I have installed SDK, WDK and can see the file IddCx.dll in WinSxS subfolders. Also there is a IddCx.h file within the root directory.

**Comment by averagejooe** [comment]
Heya,

I really like this its really helpful to get more real screen estate for remote support activity and such. I wonder if there is a possibility to force my 1080p monitor to use 1440p resolution somehow? 
 (I am currently remote working so it is a perfect time to test this utility haha)

**Comment by blakerboy777** [comment]
Is there a reason 90hz isn't available as an option on resolutions above 1920 x 1080?

**Comment by dEEkAy2k9** [comment]
I have been wondering about one thing.

I just recently got into VR stuff on PC thanks to the PSVR2 getting support for SteamVR. Now i do own a 32:9 5120x1440 super ultrawide display. Going into SteamVR, my desktop is available as a screen in the virtual reality space, either through different apps or through the dashboard. Now since i run my display at native resolution of 5120x1440, the available desktop in VR is HUGE.

While in VR, all i would actually need is a small 1920x1080 or maybe 2560x1440 resolution desktop to navigate stuff like chrome, discord or just spotify.

Would this virtual display driver enable me to just create a 1920x1080 display and use that one for the time i am in VR? I tried unplugging turning off my display but what then happens, it just picks up the TV that's connected to my pc and uses a 4k display instead for desktop stuff. I don't want to go around and unplug every display i own just to have less hassle in VR.

How does this Virtual Display behave on windows itself? I seem to find a ton of videos on youtube about the installation and setup process but no one actually shows the virtual display at all. Is it a separate window i can just use? Is it off-screen all the time?

As you can see, a lot of "strange questions" from a guy just discovering this.

**Reply by averagejooe** [reply]
> I have been wondering about one thing.
> 
> I just recently got into VR stuff on PC thanks to the PSVR2 getting support for SteamVR. Now i do own a 32:9 5120x1440 super ultrawide display. Going into SteamVR, my desktop is available as a screen in the virtual reality space, either through different apps or through the dashboard. Now since i run my display at native resolution of 5120x1440, the available desktop in VR is HUGE.
> 
> While in VR, all i would actually need is a small 1920x1080 or maybe 2560x1440 resolution desktop to navigate stuff like chrome, discord or just spotify.
> 
> Would this virtual display driver enable me to just create a 1920x1080 display and use that one for the time i am in VR? I tried unplugging turning off my display but what then happens, it just picks up the TV that's connected to my pc and uses a 4k display instead for desktop stuff. I don't want to go around and unplug every display i own just to have less hassle in VR.
> 
> How does this Virtual Display behave on windows itself? I seem to find a ton of videos on youtube about the installation and setup process but no one actually shows the virtual display at all. Is it a separate window i can just use? Is it off-screen all the time?
> 
> As you can see, a lot of "strange questions" from a guy just discovering this.

I don't have VR so i don't fully understand what you mean. Basically, your understanding of this tool is correct on the last sentence. It is a virtual display so it's off screen all the time. So for instance if I only have a full HD monitor in the office, I cannot use this tool to display  a bigger resolution like QHD or 4K, ON THE MONITOR itself.  But I can create a QHD or 4K virtual display, then when I need to access the PC remotely in my QHD monitor at home, I could pick utilize the QHD resolution set on VDD by toggling which display set as primary/extend, then access this added screen real estate using Moonlight/Sunshine/Parsec streaming. 
So, like it's mainly beneficial for headless setup. For your VR, it would depend if the software like SteamVR (sorry I don't know if this is correct or not I don't own one) can "see" the virtual display when set as primary.

If you want to try this, I recommend you to install some remote assist tool like realVNC/Parsec of some sorts, because naturally when you set this virtual display as primary, you cannot see it on your  4K TV nor 32:9 monitor, so you need another device to access this remotely.

**Reply by dEEkAy2k9** [reply]
This explanation helped a lot, thanks.
Seems like i will just have to try it out and see what happens in VR then 👍 

I might post my results here afterwards.
